Arranxar algo o modelo simplificado

deivisCode opened the pull request:
Varios erros e cousas que non iban ben. Explicoo todo en comentarios que é máis fácil

deivisCode committed:
Arranxar algo o modelo simplificado

deivisCode commented on Makefile:
```diff
@@ -26,6 +26,13 @@ limpa:
+	zip -r modelo.zip \
+		american-physics-society.csl \
+		artigo_simplificado.tex \
+		bibliografia.bib \
```
Facía falta engadir unha bibliografía. Puxen a que xa temos. Por un lado, mostra exemplos de bibliografías REAIS, pero é bastante grande, e sería máis facil que nos entregasen un .bib simple. Unha opcion é crer un directorio pequecho ca bibliografía, co artigo simplificado, etc. Así queda todo no mesmo sitio, e podemos engadir máis arquivos sen enguarrar o espazo de traballo

arebla replied:
@deivisCode Sep. Directorio `exemplo` ou algunha variación diso e metemos todo o do artigo simplificado (imaxe portada, bibliografia mínima e artigo simplificado). Outra cousa que rexe é facer que cada revista teña a súa bibliografía individual no directorio correspondente, non ten sentido ter un único ficheiro con centos de entradas.

deivisCode commented on Makefile:
```diff
@@ -26,6 +26,13 @@ limpa:
+		american-physics-society.csl \
+		artigo_simplificado.tex \
+		bibliografia.bib \
+		exemplo_imaxe.jpeg \
```
Tamén facía falta unha imaxe. Esta é a imaxe do espazo que usábamos de portada, pero cambielle o nome así é algo máis xeral

deivisCode commented on artigo_simplificado.tex:
```diff
@@ -58,8 +58,9 @@
-\documentclass{revista}
+%%%: Esto importa a clase definida no arquivo aparte, 'revista.cls'. Usamos a opción
+%%%: 'simple' porque non queremos portada, índice nin contraportada neste caso
+\documentclass[simple]{revista}
```
As opcións de clase!!

deivisCode commented on artigo_simplificado.tex:
```diff
@@ -73,29 +74,21 @@
 %%%:               actualidadeCientifica,filosofia,entrevistas,
 %%%:               programacion,pasatempos,anuncios,profesorado
+%%%: {TITULO}   Evidente, o titulo do artigo. Non debería ser moi longo.
+%%%: {PERSONA}  Autores do artigo
```
Facía falta cambiar a orde dos argumentos

deivisCode commented on artigo_simplificado.tex:
```diff
@@ -122,16 +115,24 @@ \subsection*{Introdución}
-\begin{centering}
+%%%: Podense engadir imaxes co entorno 'figure' e ca opción [H] (é obligatoria)
+%%%: Se podedes, intentade que as imaxes non sexan demasiado pesadas, porfavor
+\begin{figure}[H]
```
As figuras, inda taba esto escrito co `\begin{centering}`. Dado que o tema de engadir `[H]` é 100% obligatorio (e a xente olvidaselle), podemos facer algo como

```latex
% https://tex.stackexchange.com/a/11342
\makeatletter
    \def\fps@figure{H}
    \def\fps@table{H}
\makeatother
```

Ta por falar

arebla replied:
Podería dar problemas se alguén quere poñer imaxes fóra do multicols (imaxe que ocupe dúas columnas no comezo da páxina). Facemos fincapé nos comentarios do artigo exemplo e a esperar o mellor

deivisCode replied:
Inda así, entendo que se se termina o entorno multicols e se usa unha figura grande, tamén imos querer que teña H. Calquera outra cousa, tpbh! ou o que sexa, igual recoloca todo. Supoño que se pode probar (ben comentado todo, claro), total ten facil solución

arebla replied:
Non necesariamente, entendo que a H é para colocar *Here* segundo a posición no .tex, polo que se queres que estea no comezo da páxina [!t] ou no final [!b] terás que poder cambiar o parámetro no figure

deivisCode committed:
Cambiadas varias cousas de sitio

deivisCode commented:
O tema de facer unha bibliografía individual para cada revista vouno deixar aparte desto

deivisCode committed:
mais links e algun comentario

deivisCode commented on funcions.lua:
Non comentara nada: todas as funcions van dentro de funcions.lua (de facer máis ca unha), asique un directorio exclusivo é redundante

deivisCode commented on revista.cls:
```diff
@@ -908,6 +906,8 @@
     % Véxase: LATEX’s hook management, sec. 2.9, en lthook.pdf
     \AtBeginDocument{%
+        % Cargamos funcions de Lua
+        \directlua{dofile('funcions.lua')}
```
Non me gusta a idea de agochar esto aquí ao final de todo, prefiero cargar as cousas ao comezo... Igual se tal cambio todo o procesado de opcións ao comezo (o cal podería ser máis comodo no caso de engadír máis funcionalidade, e.g.a través de booleanos pa cargar cousas)

deivisCode committed:
C'est fini

- Makefile rematada cas rutas adecuadas para xerar o .zip
- Makefile tiña un erro polo que non xeraba o PDF. A regra 'modelo' ten
  que ser PHONY e xa ta.
- So incluir a tipografía LatinModern no modelo, total a outra
  (NerdFonts) so se usa na version completa
- Non fai falta cargar as funcions de Lua no modelo simple. Para que non
  haxa problemas, pasamos a cargar as funcións dependendo da opción de
  clase que usemos. Reorenei o codigo para cargar as cousas ao comezo (o
  cal é típico). Daba problemas porque `\At[Begin/End]Document` necesitas
  pasarlle macros xa definidos, polo que non podia usalo pa cargar
  portada, indice, etc. ao comezo (ditos macros están ao final). SOLUCION:
  uso booleanos para determinar se debo cargar ou no algo (funcions de
  Lua, portada, indice, contraportada) É un método moi flexible

deivisCode committed:
Cargar NerdFonts só na version completa

arebla committed:
errata no artigo_simplificado.tex

arebla committed:
unificacións e erratas na clase da revista

arebla committed:
cambios mínimos artigo_simplificado.tex

- Cambio autor/persoa -> autoría que é máis inclusivo
